examples/resources/hcp_webhook/resource.tf

```diff
@@ -0,0 +1,20 @@
+resource "hcp_webhook" "example" {
```

Comment:
Should we put a comment that describes what this web hook does?

` # Notify for all new versions of a Packer image being created in the project.`

Reply:
Explaining what the webhook example does is a good call out. What about setting the description attribute for this resource instead of a comment?

Reply:
I think I will add both the comment, to describe the current state of this resource, and a better generic description, because tbh "My new webhook!" is not realistic at all.

Reply:
I will actually only add it to `description` because it became repetitive information when I wrote in both places.